### DISCUSSION TITLE
(Projects beta) Possible to populate a project in bulk via existing labels?

### DISCUSSION THREAD

**Discussion by petemounce** [community/community#19643] [Projects and Issues]
I'm getting started with Projects (beta).

I have a tonne of issues across a few repositories, and those issues have some consistent labelling.

I want to import them into my new Project(beta) by label, but I haven't found a way to do that. The docs suggest that I can only do a contains-search for the title.

That'll prevent me from going further with my trial; no way am I individually importing hundreds of existing issues, or even paging through them to add them.

I would like to have the same search-ability in the import-issues pop-out as I do in the regular issue search
* by label 
    * (this is the most important to me)
* by open/closed
* by matches-text inside `title || description`
* by repository
* by author

Edit: I'd find it extremely useful to be able to "subscribe" a project to a particular query (which in my case would probably be label-driven, like `label:"area-foo"`, and have matches therefore auto-added to the project.

**Comment by joukevandermaas** [comment]
I agree with @petemounce. I cannot seriously try projects until there's a way to import my existing stuff without manually searching for individual issues. Instead of making it easier to stay organized, it just means things are going to get lost.
